### PR TITLE
Input validator in registration

### PR DESCRIPTION
**Pull request by joffe97**
Issue #63 
Added input validation to dinner, dessert, pid and contactinfo in the institution page. Input fields uses colorcodes to show if the value is valid.

### CODE REVIEW

File: GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx [hunk 1]
```diff
@@ -28,6 +50,22 @@ function RegistrationForm() {
             setLokasjon(prevLoc => ({...prevLoc,[name]:val}))
         }
 
+        const onDinnerRadioChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+            setDinnerRadio(e.target.value);
+        } 
+
+        const getDinner = () => {
```
Comment (TheseusDeus) [inline GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx:57]: getDinner() and getDessert() is not used

File: GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx [hunk 2]
```diff
@@ -28,6 +50,22 @@ function RegistrationForm() {
...
+            return dinnerRadio === "annet" ? dinnerInput : dinnerRadio;
+        }
+
+        const getDessert = () => {
+            return dessertRadio === "annet" ? dessertInput : dessertRadio;
+        }
+
+        const onDessertRadioChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (elisabethnetland) [inline GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx:65]: An idea to set the dessert input to an empty string when the radio buttons are toggled?

File: GiEnJul/ClientApp/src/components/institution/InstitutionPersonInterface.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Gender from "../../common/enums/Gender";
+
+interface IInstitutionPerson {
```
Comment (Sommerli) [inline GiEnJul/ClientApp/src/components/institution/InstitutionPersonInterface.tsx:3]: The name IInstitutionPerson seems confusing, as the properties are connected to a recipient and not a submitter?

File: GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx [hunk 4]
```diff
@@ -1,77 +1,137 @@
...
 import { useState } from 'react';
+import InputEmail from '../InputFields/Validators/InputEmail';
+import InputNotNull from '../InputFields/Validators/InputNotNull';
+import InputPhoneNumber from '../InputFields/Validators/InputPhoneNumber';
 import Location from './InstitutionLocation';
 import InstitutionPerson from './InstitutionPerson';
+import InstitutionPersonInterface from './InstitutionPersonInterface';
 
 
+let prelokasjon: string[] = [
```
Comment (Sommerli) [inline GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx:11]: Inconsistent naming Norwegian and English

**Review by TheseusDeus — APPROVED** [review]

**Pull request merged**